index.html

```diff
@@ -117,20 +117,32 @@ <h2>API</h2>
 import * as parsel from "./parsel.js";
 self.parsel = parsel; // so that people can experiment in the console
 
+let url = new URL(location);
+const selector = url.searchParams.get("selector");
+
+if (selector) {
+	selectorText.value = selector; // set default value if the selector is empty
+	url.searchParams.set("selector", selectorText.value);
+}
+
 (tryout.oninput = e => {
-	tokens.textContent = JSON.stringify(parsel.tokenize(selectorText.value), null, "\t");
-	Prism.highlightElement(tokens);
+		url = new URL(location);
+		url.searchParams.set("selector", selectorText.value);
+		history.replaceState(null, '', url);
+
+		tokens.textContent = JSON.stringify(parsel.tokenize(selectorText.value), null, "\t");
+		Prism.highlightElement(tokens);
 
 	try {
 
-		tree.textContent = JSON.stringify(parsel.parse(selectorText.value, {recursive: recursive.checked}), null, "\t");
-		tree.classList.remove("error");
-		specificity.textContent = parsel.specificity(selectorText.value);
-		Prism.highlightElement(tree);
+			tree.textContent = JSON.stringify(parsel.parse(selectorText.value, {recursive: recursive.checked}), null, "\t");
+			tree.classList.remove("error");
+			specificity.textContent = parsel.specificity(selectorText.value);
+			Prism.highlightElement(tree);
 	}
 	catch (e) {
-		tree.classList.add("error");
-		tree.innerHTML = `<details><summary>${e}</summary>${e.stack.replace(e, "")}</details>`;
+			tree.classList.add("error");
+			tree.innerHTML = `<details><summary>${e}</summary>${e.stack.replace(e, "")}</details>`;
```

Comment:
Indentation is now correctly with tabs but seems to be 1 level more indented than before?!

Reply:
I don't know what just happened. I mean it seems normal in my editor :/

Reply:
I changed it over github and it seems okay now :D